Gemfile.lock

```diff
@@ -1,54 +1,55 @@
 PATH
   remote: .
   specs:
-    optimum-energy-rubocop (0.8.1)
-      rubocop (~> 1.42.0)
+    optimum-energy-rubocop (0.9.0)
```

Comment:
A pushed a commit after running `bundle` to get the latest version bump here too.